### ISSUE TITLE
Does HomeScreen need a screen name?

### ISSUE THREAD

**Issue by chrisklus** [phetsims/joist#629]
From https://github.com/phetsims/joist/issues/627, `screen.nameProperty` is going to be uninstrumented for the HomeScreen (and possibly single screen sims since their screen names are `null`). 

It feels cleanest to do this by checking if `options.name` exists for the screen, but HomeScreen does get passed a name string on line 41 of HomeScreen.js. While `screen.nameProperty` is used for labeling the buttons on the nav bar for sim screens (in NavigationBarScreenButton), the home screen's button uses its own type (HomeButton), and does not appear to use the string on the nav bar or the home screen itself. I'm wondering if it's used somewhere for interactive descriptions, but if not can it be omitted from HomeScreen.js? Thanks!

**Comment by zepumph** [comment]
Yes, it is used as the header for that screen:

![image](https://user-images.githubusercontent.com/6856943/79388862-92621480-7f1a-11ea-8c07-5d905a47cefe.png)


I'm not sure I understand all the nuance in trying to omit that, but I feel like using `value === null` as a check on whether or not to instrument is a bit obfuscated. I'd be happy to discuss further if you would like.

**Comment by chrisklus** [comment]
@zepumph and I just chatted on zoom - some discussion points from the call:

>I'm not sure I understand all the nuance in trying to omit that, but I feel like using `value === null` as a check on whether or not to instrument is a bit obfuscated.

This could have worked out because the designers' request was to essentially not instrument this Property if the name was not visible in the sim (for screens), which is a pattern followed by every screen except the home screen. However, as @zepumph pointed out above, HomeScreen.nameProperty is appropriately being used for accessibility, but regardless, felt that using a null check is not great because a similar pattern to the HomeScreen's use case could arise for single screen sims (and consequently further break the pattern described above).

Therefore, @zepumph recommended creating an option like `instrumentNameProperty` that defaults to true and only not instrument `nameProperty` if `options.name` is null or `instrumentNameProperty` is false. Thanks! Closing.
